response/max_age_test.go

```diff
@@ -17,7 +17,8 @@ func TestMaxAgeLongCacheTime(t *testing.T) {
 		ctx := context.Background()
 		const longCacheTime = 9999
 		cfg := &config.Config{
-			CacheTimeLong: time.Duration(longCacheTime) * time.Second,
+			CacheTimeLong:                time.Duration(longCacheTime) * time.Second,
+			IsPublishExpiryOffsetEnabled: true,
```

Comment:
This is not relevant for these tests (the value of `IsPublishExpiryOffsetEnabled` does not affect the outcome of these tests), and it should be removed, to reduce noise.